Example for interacting with Google Cloud services from Kubeflow local runner (#4064)
kubeflow/website: content/en/docs/components/pipelines/user-guides/core-functions/execute-kfp-pipelines-locally.md

- Local execution does not feature optimizations and additional features such as caching, retry, etc. While these feature are important for production pipelines, they are less critical for a local testing environment. You will find that task methods like `.set_retry`, `.set_caching_options`, etc. have no effect locally.
- Local execution makes simple assumptions about the resources available on your machine. Local execution does not support specifying resource requests/limits/affinities related to memory, cores, accelerators, etc. You will find that task methods like `.set_memory_limit`, `.set_memory_request`, `.set_accelerator_type` etc. have no effect locally.

> - Local execution doesn't support authentication mechanisms. If your component interacts with cloud resources or requires other privileged actions, you must test your pipeline in the cloud.

Reviewer: Are you removing this because local execution works after authentication? If so, does it apply to any cloud environment?

           pipeline_root='~/my/component/outputs')
```

## Google Cloud Example

> In the following example we are using SubprocessRunner to interact with Google Cloud services from local machine.

Reviewer: ```suggestion
In the following example, we use `SubprocessRunner` to interact with Google Cloud services from the local machine.
```

> Before executiong this scipt you have to authenticate with [Application Default Credentials](https://cloud.google.com/docs/authentication/provide-credentials-adc). You can use following command: `gcloud auth application-default login`

Reviewer: ```suggestion
Before executing this script, you must authenticate with [Application Default Credentials](https://cloud.google.com/docs/authentication/provide-credentials-adc). You can use the following command: `gcloud auth application-default login`
```
